lib/Launcher.js

```diff
@@ -515,19 +515,211 @@ class FirefoxLauncher {
+
+      // DevTools JSONViewer sometimes fails to load dependencies with its require.js.
+      // This doesn't affect Puppeteer but spams console (Bug 1424372)
+      'devtools.jsonview.enabled': false,
+
+      // Disable popup-blocker
+      'dom.disable_open_during_load': false,
+
+      // Enable the support for File object creation in the content process
+      // Required for |Page.setFileInputFiles| protocol method.
+      'dom.file.createInChild': true,
+
+      // Disable the ProcessHangMonitor
+      'dom.ipc.reportProcessHangs': false,
+
+      // Disable slow script dialogues
+      'dom.max_chrome_script_run_time': 0,
+      'dom.max_script_run_time': 0,
+
+      // Only load extensions from the application and user profile
+      // AddonManager.SCOPE_PROFILE + AddonManager.SCOPE_APPLICATION
+      'extensions.autoDisableScopes': 0,
+      'extensions.enabledScopes': 5,
+
+      // Disable metadata caching for installed add-ons by default
+      'extensions.getAddons.cache.enabled': false,
+
+      // Disable installing any distribution extensions or add-ons.
+      'extensions.installDistroAddons': false,
+
+      // Disabled screenshots extension
+      'extensions.screenshots.disabled': true,
+
+      // Turn off extension updates so they do not bother tests
+      'extensions.update.enabled': false,
+
+      // Turn off extension updates so they do not bother tests
+      'extensions.update.notifyUser': false,
+
+      // Make sure opening about:addons will not hit the network
+      'extensions.webservice.discoverURL': `http://${server}/dummy/discoveryURL`,
+
+      // Allow the application to have focus even it runs in the background
+      'focusmanager.testmode': true,
+      // Disable useragent updates
+      'general.useragent.updates.enabled': false,
+      // Always use network provider for geolocation tests so we bypass the
+      // macOS dialog raised by the corelocation provider
+      'geo.provider.testing': true,
+      // Do not scan Wifi
+      'geo.wifi.scan': false,
+      // No hang monitor
+      'hangmonitor.timeout': 0,
+      // Show chrome errors and warnings in the error console
+      'javascript.options.showInConsole': true,
+
+      // Disable download and usage of OpenH264: and Widevine plugins
+      'media.gmp-manager.updateEnabled': false,
+      // Prevent various error message on the console
+      // jest-puppeteer asserts that no error message is emitted by the console
+      'network.cookie.cookieBehavior': 0,
+
+      // Do not prompt for temporary redirects
+      'network.http.prompt-temp-redirect': false,
+
+      // Disable speculative connections so they are not reported as leaking
+      // when they are hanging around
+      'network.http.speculative-parallel-limit': 0,
+
+      // Do not automatically switch between offline and online
+      'network.manage-offline-status': false,
+
+      // Make sure SNTP requests do not hit the network
+      'network.sntp.pools': `${server}`,
```

Comment:
```suggestion
      'network.sntp.pools': server,
```

Reply:
I would have committed this minor change myself but I don't have [push permissions to your PR branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

Reply:
Huh, that's odd. I had "Allow edits" checked.

Reply:
Glitch in the GitHub matrix